docs/en/02_Developer_Guides/00_Model/08_SQL_Select.md

```diff
@@ -31,7 +31,7 @@ $count = DB::query('SELECT COUNT(*) FROM "Member"')->value();
 
 // Through SQLSelect abstraction layer.
 $query = new SQLSelect();
-$count = $query->setFrom('Member')->setSelect('COUNT(*)')->value();
+$count = $query->setFrom('Member')->setSelect('COUNT(*)')->execute();
```

Comment:
```suggestion
$count = $query->setFrom('Member')->setSelect('COUNT(*)')->execute()->value();
```
`SQLSelect::execute()` returns a `Query` instance, from which you then get the value: https://api.silverstripe.org/4/SilverStripe/ORM/Queries/SQLSelect.html#method_execute